### DISCUSSION TITLE
cs2-WeaponPaints installation

### DISCUSSION THREAD

**Discussion by f0newy** [Nereziel/cs2-WeaponPaints#278] [Q&A]
I've been trying to do install this plugin to my dathost server for quite some time now and still couldn't. Could someone show my how?
What i've tried;
- download weaponpaints.zip and upload the custom plugin to my dathost server then reboot.
- set FollowCS2ServerGuidelines to false then reboot.

**Comment by mayersgamer** [comment]
did you setup the database ?

**Reply by uro936-sys** [reply]
I have the same problem too,I tried this,But it didn't help me
